### PR TITLE
Run ESLint

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Runs ESLint on builds.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,7 @@ NAME = storybook-${REAL_ENV}
 TAG = libero/storybook:${REAL_ENV}
 EXISTING_CONTAINERS = $$(docker ps --all --quiet --filter "name=${NAME}")
 MOUNT := $$(if [[ ${REAL_ENV} = "dev" ]]; then echo "--mount type=bind,source=$$(pwd)/src/,destination=/app/src,readonly --mount type=bind,source=$$(pwd)/.storybook/,destination=/app/.storybook,readonly"; fi)
+MOUNT_WRITE := $$(if [[ ${REAL_ENV} = "dev" ]]; then echo "--mount type=bind,source=$$(pwd)/src/,destination=/app/src --mount type=bind,source=$$(pwd)/.storybook/,destination=/app/.storybook"; fi)
```
Comment (thewilkybarkid) [inline Makefile:8]: Maybe should just not use `readonly` mounts, doesn't really add much.

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+!/.eslintrc.js
+!/.storybook
```
Comment (thewilkybarkid) [inline .eslintignore:2]: Couldn't find another way to stop dot files/dirs being ignored.

File: Makefile (hunk 1 above)
Comment (davidcmoulton) [reply Makefile:8]: 🤷‍♂

**Comment by davidcmoulton** [comment]
Like that it uses `make` as the interface to the build tasks 👍

**Review by davidcmoulton — APPROVED** [review]

**Pull request merged**